js/ui/save-html.js

```diff
@@ -75,6 +80,17 @@ define(
                 }
                 ui.freshModal("Save Snapshot", $div);
             }
+        // Create and download an EPUB 3 version of the content
+        // Using (by default) the EPUB 3 conversion service set up at labs.w3.org/epub-generator
+        // For more details on that service, see https://github.com/iherman/respec2epub
+        ,   makeEPubHref: function(){
+                if (!conf.epubGenTool) {
+                    conf.epubGenTool = "https://labs.w3.org/epub-generator/cgi-bin/epub-generator.py";
```

Comment:
> On 08 Sep 2015, at 10:35 , Tobie Langel <notifications@github.com> wrote:
> 
> In js/ui/save-html.js <https://github.com/w3c/respec/pull/487#discussion_r38901367>:
> 
> > @@ -75,6 +80,17 @@ define(
> >                  }
> >                  ui.freshModal("Save Snapshot", $div);
> >              }
> > +        // Create and download an EPUB 3 version of the content
> > +        // Using (by default) the EPUB 3 conversion service set up at labs.w3.org/epub-generator
> > +        // For more details on that service, see https://github.com/iherman/respec2epub
> > +        ,   makeEPubHref: function(){
> > +                if (!conf.epubGenTool) {
> > +                    conf.epubGenTool = "https://labs.w3.org/epub-generator/cgi-bin/epub-generator.py";
> Well, it assumes the API will be the exact same. If you really want to keep your options open, you should be providing a function that calls a callback when it's done with the processing. But then that doesn't serialize well (which might be an issue now that we're serializing the config in the target doc).
> 
> 

That is true.

To be honest, I do not really feel too strongly about this…


----
Ivan Herman, W3C
Digital Publishing Lead
Home: http://www.w3.org/People/Ivan/
mobile: +31-641044153
ORCID ID: http://orcid.org/0000-0003-0782-2704